src/AutoComplete/reactSelectThemeOverrides.js

```diff
@@ -0,0 +1,29 @@
+import { borderRadius, colors } from '../theme';
```

Comment:
Was there no way to avoid pulling in the theme directly like this? It would result in potential inconsistencies if a theme was used within the application.

Reply:
I don't think you would want to pull the theme in via provider here. You can overwrite these values via props if you need to customize them. This is simply setting the default theme utilizing arbor values at the base rather than react-select values.